src/core/Dropdown/Dropdown/Dropdown.tsx

```diff
@@ -58,6 +58,8 @@ export interface DropdownProviderState {
    * Callback for communicating DropdownItem Tab key press to the parent
    */
   onItemTabPress: () => void;
+  /** Callback for communicating DropdownItem mouse over to parent  */
+  onItemMouseOver: (itemIndex: string) => void;
```

Comment:
Should index be type number?

Reply:
The variable was actually badly named, it's not the item's index but rather the string value which is the parameter here. Changed it from `itemIndex ` to `itemValue`